docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -1,16 +1,381 @@
-# 🔧 Development using Visual Studio Code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/282) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Development using Visual Studio Code
+
+by [Bertrand Le Roy](https://github.com/bleroy),  [Phillip Carter](https://github.com/cartermp),
+[Bill Wagner](https://github.com/billwagner)
+
+Contributions by [Toni Solarin-Sodara](https://github.com/tsolarin)
+
+This document provides a tour of the steps and workflow to create a .NET
+Core Solution using [Visual Studio Code](http://codde.visualstudio.com).
+You'll learn how to create projects, create unit tests, use the debugging
+tools, and incorproate third-party libraries via [NuGet](http://nuget.org).
+
+This article uses Visual Studio Code on Mac OS. Where there are differences,
+it points out the differences for the Windows platform.
+
+Prerequisites
+-------------
+
+Before starting, you'll need to install the [.NET Core SDK](https://www.microsoft.com/net/download),
+currently in a preview relese. The .NET Core SDK includes the latest release
```

Comment:
@tdykstra I fixed the typo. I think the SDK will still be a Preview with the next update.